XXXexamples_test.go

```diff
@@ -49,25 +49,24 @@ func ExampleNewBowFromColBasedInterfaces() {
 }
 
 func ExampleNewBowFromRowBasedInterfaces() {
-	colNames := []string{"time", "value", "valueFromJSON"}
```

Comment:
Why do we replace an existing test ? Should not we need to add a new instead as we add a new feature ?

Reply:
But we are here in a test file and I imagine this function impacts one or multiple tests. Sounds weird and not a good practice to transform initial states of existing tests that bound the existing production code behaviours. I would expect to create new tests with initial states with timestamped Bows and keep the existing ones with  not timestamped Bows.

Reply:
This is really not a test, this is specific to the Go language, have a look here : https://pkg.go.dev/testing#hdr-Examples
I transformed this example to avoid having too much information when a new user arrive on the project documentation page here: https://pkg.go.dev/github.com/metronlab/bow#pkg-examples